sys/mips/mips/pm_machdep.c

```diff
@@ -180,15 +187,21 @@ sendsig(sig_t catcher, ksiginfo_t *ksi, sigset_t *mask)
 #endif
 
 	/* save user context */
-	bzero(&sf, sizeof(sf));
+	bzero(&sf, sizeof(struct sigframe));
```

Comment:
Why this change? Using the variable seems less likely to break in the future.

Reply:
Reduces diff to upstream which uses the type.